dags/populate_metabase_fluxiae.py

```diff
@@ -0,0 +1,74 @@
+import os
+import shutil
+from tempfile import TemporaryDirectory
+
+import airflow
+from airflow.operators import bash, empty, python
+from airflow.operators.trigger_dagrun import TriggerDagRunOperator
+
+from dags.common import default_dag_args, slack
+from dags.flux_iae import process_flux_iae
+
+
+dag_args = default_dag_args()
+
+with airflow.DAG(
+    dag_id="populate_metabase_fluxiae",
+    schedule_interval=None,
+    **dag_args,
+) as dag:
+
+    def create_import_directory(params=None, **kwargs):
+        kwargs["ti"].xcom_push("temp_dir", TemporaryDirectory().name)
```

Comment:
Proposition de nouveau nom :
```suggestion
        kwargs["ti"].xcom_push("work_directory", TemporaryDirectory().name)
```
- On est pas payé aux caractères économisés :grin:, et l'autocomplétion fait que ce n'est pas plus long à taper mais ça aide la lecture de ne pas avoir d’abréviation.
- "work" correspond plus à ce qu'on en fait, surtout que le fait d'utiliser un `TemporaryDirectory()` est un détail d'implémentation.
- Mon petit doigt me dit qu'on avoir besoin de cette logique à d'autre moment donc autant partir sur un nom réutilisable vu que ça coûte rien de plus.